### ISSUE TITLE
does dataset_tool.py default to 72DPI with no option for higher dpi?

### ISSUE THREAD

**Issue by MoemaMike** [NVlabs/stylegan2-ada-pytorch#75]
i am a photographer trying to train models based on my large collection of photos. Normally DPI 300 is preferred. I noticed that when run dataset_tool.py it creates png files at 72DPI from my 300DPI jpg files. that is a significant loss in quality. Is this overridable or have you decided that 72 DPI for 1024x1024 is the limt for now?

**Comment by nurpax** [comment]
The default is not to change the source image resolution.  So if you give it 1024x1024 resolution images, the output dataset should be in 1024x1024 resolution.  The dataset_tool.py does not care about DPI, it only works with pixel resolution.

**Comment by MoemaMike** [comment]
thank you. Any plans to support larger than 1024 images? You may close issue

**Comment by nurpax** [comment]
I think the code supports larger than 1024.  I don't know off the cuff if the hyperparameter choices are good for that and not sure how much GPU RAM you will need to train.
